src/lib/extListen.js

```diff
@@ -0,0 +1,32 @@
+export async function getAllExtensions() {
+  const extensions = await browser.management.getAll();
+  return extensions.filter((extension) => extension.type !== 'theme');
+}
+
```

Comment:
Do any have any suggestion regarding this filename ? (I named it `extListen.js`, as I wasn't getting any good name). This file will contain methods interacts with background script (currently). These methods can be used in both popup and extension page.

Reply:
** Update **: I renamed it to `ext-listen.js`